tests/src/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -173,31 +216,37 @@ class WskCoreBasicTests
             }
     }
 
-    behavior of "Wsk Action CLI"
+    behavior of "Wsk Package Action"
 
-    it should "get an action from package" in withAssetCleaner(otherWskProps) {
+    it should "get and invoke an action from package" in withAssetCleaner(guestWskProps) {
         (wp, assetHelper) =>
             assetHelper.withCleaner(wsk.pkg, samplePackage) {
                 (pkg, _) => pkg.create(samplePackage, parameters = Map("a" -> "A".toJson), shared = Some(true))(wp)
             }
 
             assetHelper.withCleaner(wsk.action, sampleAction) {
-                val file = Some(TestUtils.getTestActionFilename("empty.js"))
+                val file = Some(TestUtils.getTestActionFilename("hello.js"))
                 (action, _) => action.create(sampleAction, file, shared = Some(true))(wp)
             }
 
-            val fullyQualifiedActionName = s"/$otherNamespace/$sampleAction"
-            val stdout = wsk.action.get(fullyQualifiedActionName)(originWskProps).stdout
+            val fullyQualifiedActionName = s"/$guestNamespace/$sampleAction"
+            val stdout = wsk.action.get(fullyQualifiedActionName)(defaultWskProps).stdout
             stdout should include("name")
             stdout should include("parameters")
             stdout should include("limits")
             stdout should include regex (""""key": "a"""")
             stdout should include regex (""""value": "A"""")
+
+            val run = wsk.action.invoke(fullyQualifiedActionName)(defaultWskProps)
+
+            withActivation(wsk.activation, run, initialWait = 1.seconds, totalWait = 60.seconds)({
```

Comment:
Any particular reason why you need to specify a longer totalWait time? Notice that the initialWait is consistent with the default.